config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  get '/events', to: 'events#index'
+  root to: 'events#index'
+  get '/events', to: 'events#events'
```

Comment:
The route '/show' seems unnecessary. If the purpose is to show events, it would be better to use a RESTful route like '/events/:id' to show a specific event. Consider removing this route and using a show action within the 'events' controller instead.